apis/logging/v1/input_receiver_types.go

```diff
@@ -26,3 +27,11 @@ type HTTPReceiver struct {
 	// +required
 	Format string `json:"format"`
 }
+
+// SyslogReceiver receives logs from rsyslog
+type SyslogReceiver struct {
+	// Port the Service and the HTTP listener listen on.
+	// +kubebuilder:default:=10514
+	// +optional
+	Port int32 `json:"port"`
```

Comment:
any reason to support UDP?

Reply:
i will add UDP support while merging the common options in a ReceiverTypeSpec